packages/devtools_app/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"dependencyGraph":[{"name":"image_gallery_saver","dependencies":[]},{"name":"path_provider","dependencies":[]},{"name":"devtools_app","dependencies":[]}]}
```

Comment:
This file should be excluded in packages/devtools_app/.gitignore.

Reply:
@albertusangga please add it to the gitignore and remove it from the cl.